CHANGELOG.md

```diff
@@ -31,8 +31,10 @@ Here is a template for new release sections
 - pollutant and subclasses (#51)
 - structure of the ontology (#47)
 - change peat to solid and not gas (#39)
+- GeographicRegion and subclasses (#61)
```

Comment:
```suggestion
- GeographicRegion and subclasses (#62 )
```

The CHANGELOG should refer to the PR number, not the issue number

Reply:
@akleinau - you can click on "Commit suggestion", so that it will make a commit automatically :)